src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Health/StorageInitializedHealthCheckTests.cs

```diff
@@ -0,0 +1,49 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Messages.Storage;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Api.Features.Health;
+
+public class StorageInitializedHealthCheckTests
```

Comment:
The class needs a test category. The build is failing with:
Assembly 'Microsoft.Health.Fhir.R4.Api.UnitTests, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' is not compliant, because not all Test Classes have Trait 'Category'.

Reply:
Ah, oops